tools/hooks/useDayPicker.ts

```diff
@@ -0,0 +1,42 @@
+import { useState } from 'react'; // React
+import { DateUtils, RangeModifier } from 'react-day-picker'; // React DayPicker
+
+// useDayPicker inital parameters
+type useDayPicker = {
+  from: Date | undefined;
+  to: Date | undefined;
+};
+
+// useDayPicker return type
+type useDayPickerReturnType = {
+  dateRange: RangeModifier;
+  addDayToDateRange: (day: Date) => void;
+  dateRangeString: () => string;
+};
+
+/**
+ * Custom hook for managing DayPicker state
+ * @param {useDayPicker} initialDateRange - Initial date range object with from and to properties
+ * @returns dateRange state, addDayToDateRange function, dateRangeString function to display dates (YYYY-MM-DD - YYYY-MM-DD)
+ */
+export default function useDayPicker(initalDateRange: useDayPicker): useDayPickerReturnType {
+  const [dateRange, setDateRange] = useState<RangeModifier>(initalDateRange);
```

Comment:
```suggestion
export default function useDayPicker(initalDateRange?: InitialDateRange): UseDayPickerReturnType {
  const [dateRange, setDateRange] = useState<RangeModifier>(initalDateRange || { from: undefined, to: undefined });
```
We should make the initial date range optional and set the above as the default value. Following this change, we don't manually need to pass in the object with undefined `from` and `to` parameters as was done in the date range picker component.